### ISSUE TITLE
[Bug]: Installation fails in airgapped setup

### ISSUE THREAD

**Issue by 3Chris** [Icinga/helm-charts#34]
### Affected Chart

icinga-stack

### Please describe your problem

Hi,

i tried to install the chart in a airgapped setup, but it fails.

In airgapped setup you need to overwrite the repository paths and for icinga2, icingadb and icingaweb it is possible.

--set icinga2.image.repository=<local-repository>/icinga/icinga2 
--set icingadb.image.repository=<local-repository>/icinga/icingadb 
--set icingaweb.image.repository=<local-repository>/icinga/icingaweb2 

but the repository path for redis and mariadb are hardcoded at /templates/internal-databases.yaml and /templates/redis.yaml

**Comment by mocdaniel** [comment]
This is correct - in a realistic scenario you wouldn't want to rely on the integrated Redis/database solutions anyways, and rather install them externally. However, we will expose the relevant fields from the global values.yml.
